extension/observer/cfgardenobserver/README.md

```diff
@@ -20,34 +20,61 @@ The `cfgarden_observer` looks at the current host to discover Garden containers.
 ```yaml
 extensions:
   cfgarden_observer:
-    # url of the Garden socket, defaults to unix:///var/vcap/data/garden/garden.sock 
-    endpoint: my/path/to/garden.sock
     # determines how often to look for changes in endpoints.
     refresh_interval: 30s
+    # determines how often app metadata cache is refreshed
```

Comment:
Just a nit, since we have the table below that explains each configuration option, we probably don't need a comment for each option in this config example.